lib/vintage_net_wifi/wpa_supplicant.ex

```diff
@@ -544,6 +564,14 @@ defmodule VintageNetWiFi.WPASupplicant do
     )
   end
 
+  defp update_wps_creds(state) do
+    VintageNet.PropertyTable.put(
+			VintageNet,
+			["interface", state.ifname, "wifi", "wps_creds"],
```

Comment:
```suggestion
			["interface", state.ifname, "wifi", "wps_credentials"],
```

For better or worse, I've been trying to spell out words with vintage_net even though it can be a bit much to type sometimes.

Reply:
Ok, commit is on the way